tests/specs/send/send-btc.spec.ts

```diff
@@ -8,6 +8,8 @@ import { BtcFeeType } from '@shared/models/fees/bitcoin-fees.model';
 
 import { test } from '../../fixtures/fixtures';
 
+test.describe.configure({ mode: 'serial' });
```

Comment:
Can we remove? It's kinda brittle, each test should run in isolatoin

Reply:
removed